pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -118,6 +119,77 @@ func RerunDashAlertMigration(mg *migrator.Migrator) {
 	}
 }
 
+func AddDashboardUIDPanelIDMigrations(mg *migrator.Migrator) {
+	mg.AddMigration("add dashboard_uid column to alert_rule", migrator.NewAddColumnMigration(
+		migrator.Table{Name: "alert_rule"},
+		&migrator.Column{
+			Name:     "dashboard_uid",
+			Type:     migrator.DB_NVarchar,
+			Length:   40,
+			Nullable: true,
+		},
+	))
+	mg.AddMigration("add panel_id column to alert_rule", migrator.NewAddColumnMigration(
+		migrator.Table{Name: "alert_rule"},
+		&migrator.Column{
+			Name:     "panel_id",
+			Type:     migrator.DB_BigInt,
+			Nullable: true,
+		},
+	))
+	mg.AddMigration("create index to query by dashboard_uid and panel_id", migrator.NewAddIndexMigration(
+		migrator.Table{Name: "alert_rule"},
+		&migrator.Index{
+			Name: "IDX_alert_rule_org_id_namespace_uid_dashboard_uid_panel_id",
+			Cols: []string{"org_id", "namespace_uid", "dashboard_uid", "panel_id"},
+		},
+	))
+	mg.AddMigration("set dashboard_uid and panel_id from existing annotations", &setDashboardUIDPanelIDMigration{})
```

Comment:
nit: This is a backfill, that's meant to run as part of the database migrations process.
```suggestion
	mg.AddMigration("set dashboard_uid and panel_id from existing annotations", &backfillDashboardUIDPanelIDOnAlertRule{})
```

Reply:
I have renamed it to `updateDashboardUIDPanelIDMigration` as I can see all the others are called `update`. Is that OK?